lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -191,6 +199,8 @@ def add_payment_method_token(post, payment_method, options)
 
         if payment_method.is_a?(ActiveMerchant::Billing::CreditCard)
           p = create_payment_method(payment_method, options)
+          return p if p.success? == false
```

Comment:
nitpick, but might do `return p unless p.success?` here

Reply:
Cool, that does look better. Thanks again for the review, @britth